resources/frequently-asked-questions/README.md

```diff
@@ -165,6 +165,12 @@ The following settings explain how settings between the SDK and your report suit
 
 If you are unable to resolve your concerns through resources provided here, please contact [Adobe Experience Cloud customer care](https://experienceleague.adobe.com/?support-solution=General#support) for immediate assistance.
 
+## Adobe Experience Platform Edge Network
```

Comment:
Currently adding this one question in the existing general faq page, but I am open for suggestions in creating a separate faq page under the Edge extension space.

Reply:
I think this is fine for now. Once there are more FAQ items for Edge Network we can move to a new page and add a link from here to the new page.